index.html

```diff
@@ -59,11 +58,151 @@ <h1>I’m Alexandra.   Glad   to  see      you!</h1>
+            
+
+                    <section class="myproject">
+                        <div class="pheader">
+                            <img src="./img/projectthree.png">
+                            
+                        </div>
+                        <h3 class="ptitle">Tonic</h3>
+                        <uL class="details">                           
+                                
+                            <li class="conpoy"> CANOPY</li>                             
+                            <li class="backeddev">Back End Dev</li>                               
+                            <li class="text2015">2015</li> 
+                        </uL>  
+                        <h3>A daily selection of privately personalized reads; no accounts or sign-ups required.</h3>
+                        <div class="lower_part">
+                            <ul class="project-tech-stack">
+                                <li><a>HTML</a></li>
+                                <li><a>CSS</a></li>
+                                <li><a>Javascript</a></li>
+                            </ul>
+                            <a class="button">See project</a>
+                        </div>
+                    </section>
+                    <section class="myproject">
+                        <div class="pheader">
+                            <img src="./img/projectfour.png">                           
+                        
+                        </div>
+                        <h4 class="ptitle">Multi-Post Stories</h4>
+                        <uL class="details">                           
+                                
+                            <li class="conpoy"> CANOPY</li>                             
+                            <li class="backeddev">Back End Dev</li>                               
+                            <li class="text2015">2015</li> 
+                        </uL>  
+                        <h3>A daily selection of privately personalized reads; no accounts or sign-ups required.</h3>
+                    <div class="lower_part">
+                        <ul class="project-tech-stack">
+                            <li><a>HTML</a></li>
+                            <li><a>CSS</a></li>
+                            <li><a>Javascript</a></li>
+                        </ul>
+                        <a class="button">See project</a>
+                        </div>
+                    </section>
+            </section>
+            
+            
+            <section class="about-me">
+                <h2 class="titletext2">About Myself</h2>
+                <h3 class="aboutmeh3">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and 
+                    experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                </h3>
+
+
+                <h4 class="letconnect"> LET’S CONNECT</h4>
```

Comment:
- [ ] As recommended by previous reviewers of this project, please ensure that your design sticks to the template provided on Figma as close as possible as it is one of the most important reasons for this milestone  👌🏿 \